### ISSUE TITLE
change learning rate in different epoch

### ISSUE THREAD

**Issue by alevillard** [vlfeat/matconvnet#500]
Hi, how can I set to change the learning rate in matconvnet after some epochs, without stopping the training process? Thank you

**Comment by albanie** [comment]
To set a learning schedule which varies with different epochs, the learning rate can be set as an array rather than as a scalar.  In this case, the index of the value in the array corresponds to the epoch in which the learning rate is used. 

There is an example of using this approach in the imagenet example to set a log scale learning schedule [here].(https://github.com/vlfeat/matconvnet/blob/47ec58938afd468e6e53c0d549035c5a0806a26a/examples/imagenet/cnn_imagenet_init.m#L61-L65)

**Comment by alevillard** [comment]
Ok, Thank you a lot @albanie !
